work_queue/src/bindings/python3/work_queue.binding.py

```diff
@@ -1917,7 +1917,136 @@ def tree_reduce(self, fn, seq, chunk_size=2):
             seq = results
 
         return seq[0]
-           
+
+    # map, but remote
+    def Rmap(self, fn, array, name, chunk_size=1):
+        size = math.ceil(len(array)/chunk_size)
+        results = [None] * size
+        tasks = {}
+
+        for i in range(size):
+            start = i*chunk_size
+            end = start + chunk_size
```

Comment:
Let's change this one to:
```python
end = min(len(array), start+chunk_size)
```
Then you don't need the `if` below. (use the else branch)